gobblin-service/src/main/java/org/apache/gobblin/service/modules/flow/FlowGraphPath.java

```diff
@@ -142,26 +142,26 @@ private static boolean isNodeForkable(DagNode<JobExecutionPlan> dagNode) {
    */
    private Dag<JobExecutionPlan> convertHopToDag(FlowEdgeContext flowEdgeContext, Config sysConfig)
       throws SpecNotFoundException, JobTemplate.TemplateException, URISyntaxException {
-    FlowTemplate flowTemplate = flowEdgeContext.getEdge().getFlowTemplate();
-    DatasetDescriptor inputDatasetDescriptor = flowEdgeContext.getInputDatasetDescriptor();
-    DatasetDescriptor outputDatasetDescriptor = flowEdgeContext.getOutputDatasetDescriptor();
-    Config mergedConfig = flowEdgeContext.getMergedConfig();
-    SpecExecutor specExecutor = flowEdgeContext.getSpecExecutor();
-
-    List<JobExecutionPlan> jobExecutionPlans = new ArrayList<>();
-    Map<String, String> templateToJobNameMap = new HashMap<>();
-
-    //Get resolved job configs from the flow template
-    List<Config> resolvedJobConfigs = flowTemplate.getResolvedJobConfigs(mergedConfig, inputDatasetDescriptor, outputDatasetDescriptor);
-    //Iterate over each resolved job config and convert the config to a JobSpec.
-    for (Config resolvedJobConfig : resolvedJobConfigs) {
-      JobExecutionPlan jobExecutionPlan = new JobExecutionPlan.Factory().createPlan(flowSpec, resolvedJobConfig, specExecutor, flowExecutionId, sysConfig);
-      jobExecutionPlans.add(jobExecutionPlan);
-      templateToJobNameMap.put(getJobTemplateName(jobExecutionPlan), jobExecutionPlan.getJobSpec().getConfig().getString(
-          ConfigurationKeys.JOB_NAME_KEY));
-    }
-    updateJobDependencies(jobExecutionPlans, templateToJobNameMap);
-    return new JobExecutionPlanDagFactory().createDag(jobExecutionPlans);
+     FlowTemplate flowTemplate = flowEdgeContext.getEdge().getFlowTemplate();
```

Comment:
Was the formatting off for this original code block?

Reply:
Yeah it was slightly off here, the autoformat fixed it when i modified the mile